src/Drivers/ResendDriver.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace Vormkracht10\Mails\Drivers;
+
+use Illuminate\Support\Facades\Http;
+use Illuminate\Support\Facades\URL;
+use Vormkracht10\Mails\Contracts\MailDriverContract;
+use Vormkracht10\Mails\Enums\EventType;
+
+class ResendDriver extends MailDriver implements MailDriverContract
+{
+    public function registerWebhooks($components): void
+    {
+        // TODO: verify if we can hack the user endpoint to create a webhook
```

Comment:
Display a console info message using `$components` with webhook URL and the link to add this to Resend (https://resend.com/webhooks)